Source/swiftlint/Extensions/Configuration+CommandLine.swift

```diff
@@ -82,22 +83,27 @@ extension Configuration {
         return .success(groupedFiles)
     }
 
+    // swiftlint:disable:next function_body_length
```

Comment:
This function was already pretty hard to follow, now it's nearly impossible. Let's find a better way to refactor this.